src/handler.py

```
@@ -53,15 +46,22 @@ def lambda_handler(event=None, context=None, *, dry_run: bool = False):
     discord_ok = _is_configured(CONFIG.discord_token, CONFIG.discord_channel_id)
     notion_ok = _is_configured(CONFIG.notion_api_key, CONFIG.notion_database_id)
 
+    # ローカル MD 出力を先に行う: Discord/Notion で例外が出ても本文をディスクに残せる
+    md_written = False
+    try:
+        save_briefing_md(briefing, BRIEFING_OUTPUT_DIR, BRIEFING_MD_RETENTION_DAYS)
+        md_written = True
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)